car_information/label.py

```diff
@@ -0,0 +1,77 @@
+import os
+import shutil
+from car_information_service import get_data_from_vegvesenet, \
+    identify_license_plate
+import sys
+
+
+def safe_directory(directory):
+    os.makedirs(directory, exist_ok=True)
+
+
+def copy_image(image, input_directory, output_directory):
+    safe_directory(output_directory)
+    shutil.copy(input_directory + image, output_directory + image)
```

Comment:
Using `os.path.join` instead of plain concatenations removes the need to keep track of trailing/leading backslashes in parts of the path and additionally ensures compatibility across platforms.

```suggestion
    shutil.copy(os.path.join(input_directory, image), os.path.join(output_directory, image))
```